numpy/lib/function_base.py

```diff
@@ -10,14 +10,14 @@
 from numpy.core import transpose
 from numpy.core.numeric import (
     ones, zeros_like, arange, concatenate, array, asarray, asanyarray, empty,
-    ndarray, take, dot, where, intp, integer, isscalar, absolute
+    ndarray, take, dot, where, intp, integer, isscalar, absolute, normalize_axis_tuple
```

Comment:
You should break the long lines, here and below. We like lines <= 80 characters.

Reply:
@charris, thanks for your comment, should be fixed now.